<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointAddItemWidget</class>
 <widget class="QWidget" name="PointAddItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Добавить точку</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="xLabel">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="xEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="yLabel">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="yEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
